# A list of cylinders with a header.

Header header

handle_detector/CylinderMsg[] cylinders
